concepts/pattern-matching-literals/introduction.md

```diff
@@ -51,3 +51,5 @@ closerToZero n
 
 Pattern matching will test a value against each pattern from top to bottom, until it finds a matching pattern and executes the logic associated with that pattern.
 **The order of patterns matters!**
+
+[PM]: https://learnyouahaskell.github.io/syntax-in-functions#guards-guards
```

Comment:
the link text is pattern matching, not guards. It seems premature to link to the guard-guards anchor here.